src/ServerError.js

```diff
@@ -0,0 +1,26 @@
+import Parse from 'parse';
+
+/**
+ * Constructs a new ServerError object with the given message and source.
+ *
+ * The message is an error that is intended for the server logs
+ * while the source message is intended for the server logs to aid a developer
+ *
+ * @class ServerError
+ * @constructor
+ * @param {ErrorCode} code the Parse.Error code
+ * @param {String} message will be sent to clients
+ * @param {String} serverErrorLog additional information
+ *                  about the error for logging on the server
+ * @param {String} stack the stack trace
+ */
+export default class ServerError extends Parse.Error {
```

Comment:
I tried simply extending Error, but express wanted to handle it differently.  Its possible that it was another mistake, so i could go back and recheck, but i like this better anyway now.....